### DISCUSSION TITLE
Convert docs to Markdown from RST

### DISCUSSION THREAD

**Discussion by gsweats** [coddingtonbear/python-myfitnesspal#194] [General]
## TLDR

Sorry Monsieur Bear 🐻,

This Issue is sooo not focused. Please feel free to delete and I will get rid of my ramblings.

1. Thank You
2. Can I convert the `.rst` files in `/docs/` to `.md`? `Sphynx` in `build.bat` going to be ok?
3. Potential Combination of MyFitnessPal Meal Data and Glucose+Insulin history for Diabeetus Health
4. ???
5. Profit

...and some MOAR PRAISE FOR YOU 👏🙏🤓

_**READERS/OTHER ISSUE SUBMITTERS:** Please do not do as I have here. I do not know how rude it is, but the project did not have an issue template and I went crazy. Google GitHub Issue template and fill it in nice and concisely, not rambling as I have done here._

---

## Thank You / Rantings of a Fanboi
🙏 First, thank you for all you do, I love your name, and the Obsidian Local REST API (and Clipper, except I use Waterfox as my main browser now and it doesn't like that Chrome extension...hint...hint ;)).

## The Problem
It appears documentation is in `reStructuredText` (`*.rst` files in [](/docs/)).

Perhaps this is because of Sphyx or something else in the `docs/make.bat` file?

## RST over MD for a Reason?
Is there a reason for it not being `Markdown (*.md)`?
Surprising seeing as I consider you like one of several "the markdown people" out there, haha.

If there is no reason for it, I'm looking for some easy ways to up my GitHub stats.

## Potential Uses for Diabetes
As a Type 1 Diabetic, I used to utilize MyFitnessPal, and should again. Actually extracting data from it is so amazing I can't explain it. Scared to think on it as my head will fill with ideas of big data reporting tools (M$ Power BI, Google's Looker Studio, Embeddable, etc.)...too many weird things in my head as it is, lol. OH MY GOSH.... maybe I can use this, and data from Tidepool.org (upload blood sugars, insulin pump data, etc.), in order to show how different meals, perhaps even down to different items, affect blood sugar! Dude, you may say lives, thank you, Sir Adam Coddington Bear the First).

**Comment by hannahburkhardt** [comment]
Hi Greg! What's the advantage of using md over rst?
